deployment/arm/deploy-global.sh

```diff
@@ -99,18 +99,24 @@ echo "Frontend Hosts: $frontendHosts"
 echo "Backend Hosts: $backendHosts"
 echo
 
-echo "Creating global resource Group: $resourceGroupName"
+echo "######## Creating global resource Group: $resourceGroupName ########"
 az group create \
     --name $resourceGroupName \
     --location ${regions[0]}
 
+echo 
+SLEEP_DURATION=10
```

Comment:
I would prefer if we did a query & retry (max 3 times) oppose to arbitrarily sleep.